src/views/components/gp-test-results.html

```diff
@@ -155,5 +186,106 @@
 				listEl.style.height = `${itemsAmount < 3 ? avgItemElHeight * itemsAmount : avgItemElHeight * 3}px`;
 			});
 		},
+		createShareResText (urlFlow = true) {
+			if (urlFlow) {
+				// return URL
+				let origin = this.get('@global.location.origin');
+				let testResultsId = this.get('testReqParams.id');
+
+				return `${origin}/globalping?measurement=${testResultsId}`;
+			}
+
+			// return CLI cmd
+			return this.createCliCommand(this.get('testReqParams'));
+		},
+		createCliCommand (params) {
+			let cmd = Object.keys(params).reduce((cmd, key) => {
+				// test type part
+				if (key === 'type') {
+					cmd.typePart += ` ${params[key]} ${params.target}`;
+
+					if (params.locations && params.locations.length) {
+						let locationStr = params.locations.reduce((str, loc, idx) => {
+							let locPart = loc.magic.toLowerCase().trim();
+
+							if (idx === 0) {
+								return str += locPart;
+							}
+
+							return str += `+${locPart}`;
+						}, '');
+
+						cmd.typePart += ` from ${locationStr}`;
+					}
+
+					if (params.measurementOptions) {
+						let str = Object.keys(params.measurementOptions).reduce((str, key) => {
+							switch (key) {
+								case 'packets':
+									str += ` --packets ${params.measurementOptions.packets}`; break;
+
+								case 'port':
+									str += ` --port ${params.measurementOptions.port}`; break;
+
+								case 'protocol':
+									str += ` --protocol ${params.measurementOptions.protocol}`; break;
+
+								case 'type':
+									str += ` --type ${params.measurementOptions.type}`; break;
+
+								case 'trace':
+									if (params.type.toLowerCase() === 'dns' && params.measurementOptions.trace) {
+										str += ' --trace';
+									}
+
+									break;
+
+								case 'resolver':
+									str += ` --resolver ${params.measurementOptions.resolver}`; break;
+
+								case 'request':
+									if (params.measurementOptions.request.host) {
+										str += ` --host ${params.measurementOptions.request.host}`;
+									}
+
+									if (params.measurementOptions.request.path) {
+										let path = params.measurementOptions.request.path;
+
+										path = path[0] === '/' ? path : `/${path}`;
+										str += ` --path ${path}`;
+									}
+
+									if (params.measurementOptions.request.query) {
+										str += ` --query "${params.measurementOptions.request.query}"`;
```

Comment:
Use single quotes inside and add for `path` as well.

Reply:
Done